src/graphnet/models/task/classification.py

```diff
@@ -1,9 +1,33 @@
 """Classification-specific `Model` class(es)."""
 
-import torch
-from torch import Tensor
+from torch import Tensor, sigmoid
+from torch.nn import Softmax
+from graphnet.utilities.config.model_config import save_model_config
 
 from graphnet.models.task import Task
+from typing import Any
```

Comment:
Group imports as (A) builtin, (B) installed, (C) within-package. Sort imports alphabetically.

```suggestion
from typing import Any

import torch
from torch import Tensor

from graphnet.models.task import Task
from graphnet.utilities.config.model_config import save_model_config
```

Reply:
Makes sense, I will do this moving forward 🙂